### DISCUSSION TITLE
Translated IP

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#7230] [Unsupported Versions] [answered]
Hello, we have a issue where there is a heavy node that is on the other side of a VPN tunnel. The VPN tunnel has NAT translation in the tunnel, so on our side of the tunnel, the box's translated IP is 10.0.0.2, but on the box itself, the IP is 192.168.0.2. When we try to login to kibana, we can't because it redirects us to the box's local IP of 192.168.0.2 instead of the translated address 10.0.0.2 that we initially accessed it on. Is there any way to resolve this?

**Comment by tomjones1977** [comment]
Can you configure it to be accessed by hostname?  That way you can set up a DNS entry to point to 10.0.0.2.

**Reply by its-a-me-mario-the-og** [reply]
It is still redirecting us and not allowing us to get to the web interface. Is there anything else we can do?

**Comment by dougburks** [comment, marked as answer]
Per @tomjones1977 's suggestion, have you followed the steps at https://docs.securityonion.net/en/2.3/url-base.html and changed the `url_base` to a hostname?
